File: 1832. Check if the Sentence Is Pangram/solution.py
class Solution:
    def checkIfPangram(self, sentence: str) -> bool:

        isCharUsed = [False]*26
        
        for c in sentence:
            isCharUsed[ord(c)-ord('a')] = True
        
        return all(flag is True for flag in isCharUsed)
        #return len (set (sentence)) == 26 one liner using set

File: 1832. Check if the Sentence Is Pangram/Solution.cs
public class Solution {
    public bool CheckIfPangram(string sentence) {
        
        var isCharUsed = new bool[26];
        
        foreach (char c in sentence) isCharUsed[c-'a'] = true;
        
        return isCharUsed.All(c => c);
     }
}
